lib/reports/estimate.rb

```diff
@@ -87,15 +89,16 @@ def total_estimated_ic_cost
     private
 
     def count_matching_items(cluster)
-      @clusters_seen << cluster.ocns.hash
+      ocns_seen.merge(cluster.ocns)
 
       @num_items_matched += cluster.ht_items.count
       cluster.ht_items.each do |ht_item|
-        if ht_item.access == "allow"
+        if !Clusterable::HtItem::IC_RIGHTS_CODES.include?(ht_item.rights)
```

Comment:
I think this would read better reversed & without the negation:

```ruby
if Clusterable::HtItem::IC_RIGHTS_CODES.include?(ht_item.rights)
  @num_items_ic += 1
else
  @num_items_pd += 1
  next
end
```